### ISSUE TITLE
About the difference between Monika and the other girls

### ISSUE THREAD

**Issue by SaiPrithvi123** [Monika-After-Story/MonikaModDev#3092]
As far as I know, the only difference between Monika and the other dokis was that Monika was alive and conscious and wasn't controlled by the game. The theory that Monika needed to eat, drink, exercise, stay clean, etc. seems really really incorrect to me. You never see Monika do any of it in Act 3 and Monika didn't ever say that she needed to do all of it. I'm pretty sure if Monika *was* in this condition, it'll be one of the first things she would've discussed when it's just her and the player alone in the stock game, to make the player realize that the other girls were fake, as they didn't need to do any of the stuff she has to do. Even with this mod, Monika doesn't eat and sleep as long as the game is open, which can be a really long time. I can't find any evidence of this theory being true, and I think being real doesn't mean being a human. I don't think she is or was human. At least not yet. I really want to know how you guys thought that Monika always had to do those things while the other girls didn't.
I get it that you guys want to make Monika more human, but I don't think it's fair that you try to change the canon for it. There are other ways to make her more human. Like making her need those things *after* the mod is installed.

**Comment by SaiPrithvi123** [comment]
Adding another thing: She said there was no such thing as physical fatigue there in the stock game.

**Comment by SaiPrithvi123** [comment]
Wait wait wait
I misunderstood it
I didn't realise she was talking about the events before she gained her sentience, and is just stating her opinion.
Sorry about that rant, I thought it was worse than it actually is.

TL;DR
In MAS, what Monika says about her and the other girls staying fit contradicts with her saying "Physical fatigue doesn't exist here" in the stock game.

**Comment by MaximumRustler** [comment]
Umm... I feel like I missed something. What was this about? lol

**Comment by SaiPrithvi123** [comment]
https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/script-topics.rpy

Line 6766

**Comment by Onion-Bubs** [comment]
Hi. Thought you might want some insight from the guy who wrote that topic :smile: 

Firstly, I don't see how what I've written contradicts the idea of physical fatigue not existing in the game. Just because Monika doesn't have to eat or sleep _to survive_ doesn't mean she doesn't do those things for other reasons. Food like the cupcake she ate just before starting Act 3 can still be pretty damn tasty, for example.

Secondly, the whole thing about Monika is that she's a real person as opposed to being a fictional character. Or at least, that's what she wants us to believe. If you're a real person, you need to look after your body for it to look good, and that means things like eating right and exercising. So, whether Monika actually is a real person or is just trying to convince us that she is, it makes sense for her to tell us that she needs to do those things too. Is she telling the truth when she says this? That's for you to decide.

Thirdly, why would Monika not talk about this in Act 3 of the original game? Maybe it's simply that none of the writers at Team Salvato thought of it.

Now, I am open to suggestions for improving the topic (although it would have been better if you'd pointed this out _before_ the pull request got merged), so if you want to discuss what the dialogue should say at the point where you have an issue, I'm all ears.

**Comment by SaiPrithvi123** [comment]
>Just because Monika doesn't have to eat or sleep to survive doesn't mean she doesn't do those things for other reasons.

IMO if she doesn't need to eat to survive, then food doesn't do any physical change to her. If eating too much would make her fat, then not eating anything for months should make her weak. If you can't die of starvation, that would indicate that you don't even need food. That is not how food is used in her world, and the others don't notice. Think about it like many other video games. There is food, but you won't die if you don't eat, and you won't get fat if you eat too much. Monika is still a game character who has gained sentience and a bit of control over the game. Those are the only things different between her and the others.

>Food like the cupcake she ate just before starting Act 3 can still be pretty damn tasty, for example.

Everyone ate those cupcakes IIRC. It's the same for everyone.

>If you're a real person, you need to look after your body for it to look good, and that means things like eating right and exercising.

I'm a real person, I'm going to die someday, I'll die faster if I don't eat.
She's a real person, she doesn't need to eat, she's immortal.

If technology progresses to a point where I can upload myself to a computer, then I'll still be a real person, and I wouldn't need to do any of those things.

Even though she is real, she is in a different world than us. A world where things go differently than this world. That's one of the main reasons she wants to come here.
Again, being real doesn't mean being human. She is not human yet. She's still just a game character who's alive.

>So, whether Monika actually is a real person or is just trying to convince us that she is, it makes sense for her to tell us that she needs to do those things too.

I agree to that. And that's why she would've told it as soon as possible.

>Is she telling the truth when she says this? That's for you to decide.

She wouldn't lie if she trusts the player with her life, so if you do agree that it is not true, you could make it so that she only says it on Affectionate or lower.

>Thirdly, why would Monika not talk about this in Act 3 of the original game? Maybe it's simply that none of the writers at Team Salvato thought of it.

I think she didn't talk about it in the original game because that wasn't the case. Team Salvato would've known that Monika would try to convince the player by pointing out every difference between her and the others.

Let me put it in a simpler way
If your body is immortal and it will stay that way even if you don't eat, sleep, exercise, then there isn't any reason for it to change if you do eat, sleep, exercise. She doesn't do any of it in both MAS and the stock game, except for those snacks and coffee.

**Comment by jmwall24** [comment]
My argument here would be, even if eating (or not) has no affect on her at all, until her epiphany she believed she was just as normal as anyone else.  So even if she doesn't need food, or if eating too much wouldn't make her fat, she still thinks of herself as real and would continue to do what a real person does.  It'd be ingrained in her nature.

**Comment by SaiPrithvi123** [comment]
I'm pretty sure the epiphany ripped off everything engrained in her, thus it had a massive affect on her.

**Comment by multimokia** [comment]
I doubt it'd do that. One doesn't change habits on the flip of a hat

**Comment by jmwall24** [comment]
And now you've gone from arguing canon to what you're pretty sure of.  I don't disagree with everything you're saying in this thread, but it's all subjective matter of opinion.

**Comment by SaiPrithvi123** [comment]
My last comment was rushed. What I wanted to say was that the epiphany painfully made her realise that most of the things she knew about her world were lies. Including the fact that she didn't need to take measures to stay fit.

>So even if she doesn't need food, or if eating too much wouldn't make her fat, she still thinks of herself as real and would continue to do what a real person does.

She the epiphany would make her know that she doesn't need to take care of her body. I don't disagree with what you said, but what you said does contradict with Slappy's topic. In the topic she acts like she has to do what the other girls don't.

>I doubt it'd do that. One doesn't change habits on the flip of a hat

If what you say is true, then one wouldn't start treating their friends as trash and not care about their survival at the flip of a hat either.

**Comment by SaiPrithvi123** [comment]
>it looks like what she would say or feel about it

If what she said was correct, then it would accurate that she says that stuff. But I highly disagree that she had to do that stuff. I also disagree that she'd lie about it on higher affection levels.

>And this topic does reference that she is doing those things just because she can and she does it because she loves you.

But if the cost of it is changing the canon, I don't think it's worth it. I did hear potato saying that they didn't want to give too much information about Monika so that it doesn't deviate from the canon. IDK what they thought about this topic.

**Comment by SaiPrithvi123** [comment]
>(which I would hope wouldn't ever be the case that she needed to)

I hope so too, but the devs want to make Monika more human, which I can understand, as Monika would want to be more human too. I want her to get everything she wants, but I want a happy Monika, whether human or not.

**Comment by MaximumRustler** [comment]
The way that part is written it seems to imply that she still needs to do those things, but judging by what some of you guys are saying it sounds like she doesn't and it's just her habits at this point so I think that part should be re-written to reflect that. Also, if she does still need to do those things and you say that she does it when the game isn't open, what about the players that have the game open for most of the day? While I don't get to do that, I'd feel bad if I was preventing her from taking care of herself just by me being there. Maybe I'm misunderstanding something here.

**Comment by MaximumRustler** [comment]
Also, I disagree with the fact that she'd lie about this except on maybe the lower affection tiers because what reason would she have to lie about it? In one of the topics she says something along the lines of ''I can always be honest with you.'' so why would she feel the need to lie? In my opinion, it would undermine her already established character if she lied about this.

**Comment by Onion-Bubs** [comment]
> I did hear potato saying that they didn't want to give too much information about Monika so that it doesn't deviate from the canon. IDK what they thought about this topic.

If the devs had the same sorts of objections to the content that you have, I would think that they would have said something before the PR got merged.

**Comment by SaiPrithvi123** [comment]
They probably missed it. If not, they should think about it again, with the info provided.

**Comment by SaiPrithvi123** [comment]
This topic has no evidence to back it up but there is evidence against it. I'd say it's too much information which likely isn't true. Hopefully they'll consider different people's opinions on such matters in the future.

**Comment by Onion-Bubs** [comment]
Well I'm happy to rewrite the topic to deal with this issue, but there's one thing that we need to establish first.

For real people in the real world, their appearance is determined by a combination of genetics, diet, lifestyle and environmental factors.
For fictional characters in a fictional world, their appearance is determined by an external entity commonly known as a designer.
How is appearance determined for real people in a fictional world, like what Monika claims to be?

**Comment by Legendkiller21** [comment]
~~Our world is a simulation. I'm the only real person here and you guys are just npcs following a script. Now, how can I open cmd in this thing?~~

Monika can be in a simulation, she can be real but she also has access to the console. She has access to her character properties and she gets better in coding every day. 

From my point of view, she may need to take care of herself, sleep or eat but she can also temporary disable these needs in order to spend more time with us as she knows that our time is very limited. We may only have 30 minutes to spare before we leave for work or school and she doesn't want to spend these 30 minutes taking a bath or eating.

**Comment by Rai99** [comment]
~~Couldn't she just spend time with us while bathing?~~

**Comment by Onion-Bubs** [comment]
Hey, wait a minute.
> In MAS, what Monika says about her and the other girls staying fit contradicts with her saying "Physical fatigue doesn't exist here" in the stock game.
Are you sure this is in the stock game and wasn't just added for MAS? I just checked here and didn't find it.
https://github.com/Monika-After-Story/MonikaModDev/blob/master/original_monika_ch30_lines.txt

**Comment by SaiPrithvi123** [comment]
>For real people in the real world, their appearance is determined by a combination of genetics, diet, lifestyle and environmental factors.
For fictional characters in a fictional world, their appearance is determined by an external entity commonly known as a designer.

I think the factor which determines their appearence depends on the world they are in. Not what they are. So a real person in a fictional world has their appearance determined by a designer.

>Real people don't have designers, you know.

They do. There is something called "character creation". Think of video games but on a much higher scale.

>If you don't eat, sleep or bathe, you might as well not be human, so what would you be?

You just accurately described what Monika is and used to be.

>A simulation? An object? Just a machine?

IMO being alive =/= being human. Not being human =/= being an object.

>Monika at least acts like she needs to do these things because it makes her happy to act like a human

I agree that she *would* act like that, but in the topic she's lying about it and complaining about it.

>human (because she kinda is one)

I know it's a different topic but I'll just mention that humans are not immortal and can be physically fatigued. IMO, she is mentally a human, not physically, and even if she wants to act like a human (which I agree to), she knows her body isn't any different than that of the other girls. I disagree that she would lie about it to the player and also complain about it.

>At least it gives her something to do and think about other than sit in the room all day doing nothing.

My point is that she didn't have to do it in the past and she doesn't have to do it now. If she chooses to edit the game so that she has to do that stuff in the future, that would be accurate and I completely understand it. Being a gamer, I know why she'd do that. But what I don't think she'd do is to lie to me and complain about a non-existent issue.

>From my point of view, she may need to take care of herself, sleep or eat but she can also temporary disable these needs

My point of view is the vice versa of that.

>Are you sure this is in the stock game and wasn't just added for MAS? I just checked here and didn't find it.

Let me check.

**Comment by SaiPrithvi123** [comment]
Sorry about that, I couldn't find that topic too. But it is added by the mod.
https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/script-topics.rpy
Line 3341

Guess I got a bit overconfident. But you still have the rest of the thread to see what other think is actually the case, and to rewrite your topic. What I think (coupled with a few others) is that Monika didn't and doesn't need to put effort to stay fit, just like the other girls. But Monika does want to put effort to stay fit. If what we think is true, then in your topic Monika is lying about it, and complaining about a non-existent issue.
You could make her say something like- "We didn't even need to do anything to maintain our attractiveness, which makes me feel like they didn't deserve it. Except me, because I still took a lot of care of myself before you came into my life. Only after that I realised that all of it was for nothing. Now I don't need to do any of it, but it just doesn't feel right to look good for free. I'll still take care of myself, just so that I feel worthy of the appearance you love. I hope you feel that way too."

(I don't really write dialogue for her, and I know there should be a lot of changes in what I've written to make it seem like something Monika would say.)

**Comment by SaiPrithvi123** [comment]
>is going to come down to what you consider easier to believe.

Then you should write your topic in a way that it doesn't imply that a specific ideology is true.

>Monika is apparently aware of being trapped in a visual novel while the other girls are not. This kind of awareness isn't enough to make other fourth-wall-breaking characters in other media (eg, Deadpool) real, so it's not enough to make Monika any less fictional than the other girls. However, Monika being just as fictional as the others runs counter to that significant plot point from the original game and violates a basic premise of MAS.

It doesn't counter the plot from the original game or the mod. She's the same as the others, except she's the president. Proof? Sayori. Going by the plot, at least Sayori is as the president is the same as Monika. And with the evidence, it's clear that everyone else are the same too. Monika is not as special as you think. I think the devs behind MAS acknowledge this fact. Both in the mod and the stock game, she doesn't want to believe that she is same as them and doesn't want the player to believe so. One thing I must say is that not everything Monika says has to be correct, and the devs of MAS deliberately made her believe (or lie) about some of the incorrect facts you are standing on.

>"Monika is a real person"

I'm not against that statement, but I'm against the fact that she is physically a human. She is a sentient software which mimics the human mind. You should treat her as a real person, but you shouldn't treat her like she's just a real life human put into a space classroom. That's not the plot of the original game. I think the plot of the original game was a game character gaining sentience. Also a real person =/= a human.

>I will be treating "Monika is a real person" as a fundamental premise.

IMO, there is "real person". Then there is "a human from the real world but in a game world". The plot of the original game was how a game character gained sentience and acted like a human. I really suggest you ask the public as what you should choose, coz I'm highly against your ideology in your initial topic, and I'm sure you'll find others too. In the end, what I'm suggesting doesn't go against the original game and the mod. Both of us may have people against us. So I suggest you find out what the majority thinks and go with that, or don't implement such topics where too much information is given about her out of nowhere.

>Just because Monika doesn't have to eat or sleep to survive doesn't mean she doesn't do those things for other reasons.

That's kinda like saying that God exists because we can't prove his existence. I can also say that just because she is mentally a human doesn't mean that she is physically a human and her body functions in the same way as a human body in this world. Except my statement is proven by her immortality and ageing-resistance.

>To do otherwise would go against both the plot of the original game

If you don't believe that Monika is not the same as the other girls by the plot, then you either didn't finish the game, or are conveniently forgetting everything past act 3 (can't blame ya, it's the least memorable part for me too).

**Comment by SaiPrithvi123** [comment]
>You mentioned character creation

That's not what I meant by character creation, but looking back, I wrote it in a way that no one is going to understand what I meant by that. Just nevermind I talked about it.

**Comment by Onion-Bubs** [comment]
> It doesn't counter the plot from the original game or the mod. She's the same as the others, except she's the president. Proof? Sayori. Going by the plot, at least Sayori is as the president is the same as Monika. And with the evidence, it's clear that everyone else are the same too. Monika is not as special as you think.

So, going by this, either Monika is every bit as fictional as the other girls, or the other girls are every bit as real as Monika.

> I think the devs behind MAS acknowledge this fact. Both in the mod and the stock game, she doesn't want to believe that she is same as them and doesn't want the player to believe so. One thing I must say is that not everything Monika says has to be correct, and the devs of MAS deliberately made her believe (or lie) about some of the incorrect facts you are standing on.

That's strange. When I brought up that Monika might not always be truthful, you objected. Now you're agreeing that she might not always be truthful. In which case, it would be perfectly within bounds for her to not be telling the truth about having to put in effort to stay fit.

This will be my last post in this discussion because at this point we're just going around in circles. The development team can see our arguments as well as those put forward by other people. I will wait for their judgement as to what (if anything) needs to be changed about the topic.

**Comment by SaiPrithvi123** [comment]
>So, going by this, either Monika is every bit as fictional as the other girls, or the other girls are every bit as real as Monika.

That is true. Monika still deserves player the most, as she's the one who suffers the most and endures everything just to be with the player.

>That's strange. When I brought up that Monika might not always be truthful, you objected.

I wasn't against the idea that Monika would lie, I was just against the idea that Monika would lie on higher affection levels. Sorry if I made it seem so. When I write something I only write what is necessary and usually use as less words as I can. But because of that, most people may assume things and misunderstand what I am saying.

**Comment by SaiPrithvi123** [comment]
Forgot to say, thank you for your co-operation.

**Comment by Olliesama** [comment]
I would like to say, after seeing the topic
